### PR TITLE
Add errors_confirm API

### PR DESCRIPTION
**Pull request by ylukin**
**Description**
This PR proposes adding support for `/mowers/{id}/errors/confirm` API endpoint to reset non-fatal mower errors. 

**Why it's important**
Automower can go into ERROR state for any number of reasons. Certain non-fatal errors can be reset remotely (see [example](https://www.youtube.com/watch?v=0UjBZm2DvX4#t=1m38s)) via Husqvarna app so the mower can continue with it's schedule. For example, I use [this gate](https://store.smart-dots.com/product-details/626c6f0d05b0ca664defa2f1) with my Automower and occasionally the mower fails to get through the gate, thereby going into ERROR state. Most of the time, I can reset the error via Husqvarna app and resume schedule. I'd like to do this through automation to minimize human intervention. 

**Next Steps**
Add support for "Reset Error" call in [husqvarna_automower](https://www.home-assistant.io/integrations/husqvarna_automower/) Home Assistant integration that depends on this aioautomower library.

### CODE REVIEW

**Comment by Thomas55555** [comment]
Thanks. Some checks are failing, and also a label. Can you add this? And can you also add tests, like here?
https://github.com/Thomas55555/aioautomower/blob/2c414860c475429ac796318f127710e5a09c5d9c/tests/test_session.py#L76-L81

File: tests/test_session.py [hunk 1]
```diff
@@ -97,6 +97,12 @@ async def test_connect(
         f"mowers/{MOWER_ID}/calendar",
         json={"data": {"type": "calendar", "attributes": {"tasks": task_list}}},
     )
+    await automower_api.error_confirm(MOWER_ID)
+    assert mocked_method.call_count == 10
+    mocked_method.assert_called_with(
+        f"mowers/{mower_id}/errors/confirm",
```
Comment (Thomas55555) [inline tests/test_session.py:103]: You must use the `MOWER_ID` constant. In capital letters.

Comment (ylukin) [reply tests/test_session.py:103]: Yeah I just saw...fixing

**Comment by ylukin** [comment]
I added test as requested. I am unable to add labels to this PR as I do not have permissions (probably need to be a collaborator on your repository to add labels).

**Comment by Thomas55555** [comment]
I've added the label. You can disable the pylint warning and just take care of the mypy failure.

**Comment by ylukin** [comment]
Added type hint to fix the mypy issue, don't think I have ability to disable other checks (e.g. pylint) in your repository.

**Comment by ylukin** [comment]
Thanks, I think that took care of all remaining issues now, please take a look.

**Comment by Thomas55555** [comment]
Tanks

**Pull request merged**